backend/app/processing/vanna_integrator.py

```
@@ -0,0 +1,267 @@
+from core.logger import logger
+
+# Use your custom subclass that combines ChromaDB vector store + Gemini chat
+from vanna.chromadb import ChromaDB_VectorStore
+from vanna.google import GoogleGeminiChat
+# PersistentClient to make Chroma auto‑persist into our folder
+from chromadb import PersistentClient
+from chromadb.config import Settings
+
+
+class MyVanna(ChromaDB_VectorStore, GoogleGeminiChat):
+    def __init__(self, config: Dict[str, Any] = None):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)